contracts/WitnetRequestsBoard.sol

```diff
@@ -164,7 +165,7 @@ contract WitnetRequestsBoard is WitnetRequestsBoardInterface {
 
     // Create a new `DataRequest` object and initialize all the non-default fields
     DataRequest memory request;
-    request.dr = _serialized;
+    request.requestAddress = _requestAddress;
```

Comment:
We should store the hash of the DR to avoid potential attacks, as     uint256 drOutputHash = uint256(sha256(requestContract.bytecode())). Then we should use this value when the PoI is demonstrated